specs/Deployment/DeploymentAPI.md

```diff
@@ -155,6 +202,15 @@ namespace Microsoft.Windows.ApplicationModel.WindowsAppRuntime
         HRESULT ExtendedError{ get; };
     };
 
+    /// This object is used to specify deployment options to apply when using DeploymentManager's
+    /// Initialize method
+    runtimeClass DeploymentOptions
+    {
+        /// Gets or sets a value that indicates whether the processes associated with the package will be
+        /// shut down forcibly so that installation of the packages can continue if they are currently in use
+        bool ForceTargetApplicationShutdown{ get; set;};
```

Comment:
NIT: `bool ForceTargetApplicationShutdown;`

If you don't specify get and/or set then the default is you have both